src/utils/axios.js

```diff
@@ -1,19 +1,28 @@
 import axios from 'axios'
-
-localStorage.setItem(
-  "token",
-  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiJhMjU5MWVlMC1mMzM4LTQ3NTItYTgxMi03NmFlOGY2ODg2MzciLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ._d2ul037FUeEO14s9dI9zS26QKJ6tzggYc1Og_n2pX4"
-);
-const encodedToken = localStorage.getItem('token')
-
 export const publicInstance = axios.create({
   baseURL: '/api/',
   timeout: 1000,
   headers: {'Accept': '*/*'}
 });
 
+publicInstance.interceptors.request.use(async req => {
+  console.log("intercerper",req)
+  return req
+})
+publicInstance.interceptors.response.use(async (res) => {
+  console.log("intercerper response", res);
+  return res;
+});
 export const privateInstance = axios.create({
-  baseURL: '/api/',
+  baseURL: "/api/",
   timeout: 1000,
-  headers: {'Accept': '*/*',authorization: encodedToken}
+  headers: { Accept: "*/*", authorization: localStorage.getItem("token") },
+});
+privateInstance.interceptors.request.use(async (req) => {
+  console.log("intercerper", req);
+  return req;
+});
+privateInstance.interceptors.response.use(async (res) => {
+  console.log("intercerper response", res);
+  return res;
```

Comment:
Remove interceptors if you are not doing anything.

Reply:
Done